### PR TITLE
USWDS-Site: Documentation updates for release 3.8.0 

### LINKED ISSUE #2477: USWDS-Site: Documentation updates for release 3.8.0
# Summary
Changelogs for all updates in the USWDS 3.8.0 release.

## Todo

- [x] Button group changelog
- [x] Validation & Text input changelogs
- [x] Update preview links
- [x] Add scrollable table demo fix
- [x] Remove bumped features

### Documentation page template

- Preview: [Changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/jm-changelog-doc-template/templates/documentation-page/#latest-updates)
- PR: uswds/uswds#5681
- Changelog: #2495

### Checkbox

- Preview: [Changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-changelog-5713/components/checkbox/#latest-updates)
- PR: uswds/uswds#5713
- Changelog: #2482

### In-page nav

- Preview:
   - [Documentation](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-docs-5444/components/in-page-navigation/#in-page-navigation-initialization-properties)
   - [Changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-docs-5444/components/in-page-navigation/#latest-updates)
- PR: uswds/uswds#5444
- Changelog: #2252

### Button 

- Preview:
   - [Changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/jm-enhancement-button-icons/components/button/#latest-updates)
   - [Settings entry](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/jm-enhancement-button-icons/documentation/settings/#component-settings:~:text=Icon%20gap,and%20svg%20icon.)
   - [Settings changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/jm-enhancement-button-icons/documentation/settings/#changelog)
- PR: uswds/uswds#5398
- Changelog: #2207

### Button group 

- Preview: [Changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/jm-changelog-button-group/components/button-group/#latest-updates)
- PR: uswds/uswds#5657
- Changelog: #2498

### Datepicker 

- Preview: [Changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-changelog-5701/components/date-picker/#latest-updates)
- PR: uswds/uswds#5701
- Changelog: #2483

### Validation and Text input

- Preview: 
   - [Validation](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/jm-changelog-validation-text-input/components/validation/#latest-updates)
   - [Text input](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/jm-changelog-validation-text-input/components/text-input/#latest-updates)
- PR:  uswds/uswds#5233
- Changelog: #2497

### Table

- Preview:
   - [Documentation](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-documentation-5420/components/table/#sticky-table-headers)
   - [Changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-documentation-5420/components/table/#latest-updates)
   - [Settings](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-documentation-5420/documentation/settings/#tables)
   - [Settings changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-documentation-5420/documentation/settings/#changelog)
- PR: uswds/uswds#5420
- Changelog and documentation: #2229

#### Scrollable examples

- Preview:
  - [Changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/jm-changelog-table-demo/components/table/#latest-updates)
  - Changelog PR: uswds/uswds-site#2501

### Form

- Preview: [Changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-disabled-color-contrast-changelog/documentation/settings/#changelog) 
- PR: uswds/uswds#5455
- Changelog: #2254

### Footer

- Preview: [Changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-changelog-5675/components/footer/#latest-updates)
- PR:  uswds/uswds#5675
- Changelog: #2484

## No documentation changes needed

- USWDS - Dependencies: update gulp-mocha
   - PR: https://github.com/uswds/uswds/pull/5680
   - Changelog: N/A - Dependency bump; no user-facing change
- USWDS - Dependencies: Remove experimental module work from core
   - PR: https://github.com/uswds/uswds/pull/5673
   - Changelog: N/A - Dependency bump; no user-facing change
- ~USWDS: Remove outdated normalizations~
   - ~PR: https://github.com/uswds/uswds/pull/5555~
   - ~Changelog: N/A - No user-facing change~

### PR DESCRIPTION
**Pull request by amyleadem**
# Summary
- Added changelogs and other documentation updates related to the USWDS 3.8.0 release.
- Added HTML Proofer fixes
- Updated `package-lock.json` to remove snyk error related to `SNYK-JS-ES5EXT-6095076` 

## Related issue
Closes #2477 

## Preview links

[Homepage →](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.8.0/) 

- [Button changelog ](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.8.0/components/button/#latest-updates)<br/> [Settings](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.8.0/documentation/settings/#button) <br/> [Settings changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.8.0/documentation/settings/#latest-updates)
    - uswds PR: uswds/uswds#5398
    - Changelog: #2207
- [Button group changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.8.0/components/button-group/#latest-updates)
    - uswds PR: uswds/uswds#5657
    - Changelog: #2498
- [Checkbox changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.8.0/components/checkbox/#latest-updates)
    - uswds PR: uswds/uswds#5713
    - Changelog: #2482
- [Datepicker changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.8.0/components/date-picker/#latest-updates)
    - uswds PR: uswds/uswds#5701
    - Changelog: #2483
- [Documentation page changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.8.0/templates/documentation-page/#latest-updates)
    - PR: uswds/uswds#5794
    - Changelog: #2495
- [Form changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.8.0/components/form/#latest-updates)<br/> [Settings changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.8.0/documentation/settings/#latest-updates)
    - uswds PR: uswds/uswds#5455
    - Changelog: #2254 
- [Footer changelog ](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.8.0/components/footer/#latest-updates)
    - PR:  uswds/uswds#5675
    - Changelog: #2484
- [In-page navigation changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.8.0/components/in-page-navigation/#latest-updates) <br/> [In-page navigation documentation](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.8.0/components/in-page-navigation/#in-page-navigation-initialization-properties)
    - uswds PR: uswds/uswds#5444
    - Changelog: #2252 
- [Table changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.8.0/components/table/#latest-updates) <br/> [Table settings](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.8.0/documentation/settings/#tables) <br/> [Settings changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.8.0/documentation/settings/#latest-updates)
    - Sticky table
        - uswds PR: uswds/uswds#5420
        - Changelog and documentation: #2229
    - Scrollable table example
        - uswds PR: uswds/uswds#5783
        - Changelog: uswds/uswds-site#2501
- [Text input changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.8.0/components/text-input/#latest-updates) <br/> [Validation changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.8.0/components/validation/#latest-updates)  
    - PR:  uswds/uswds#5233
    - Changelog: #2497

## No documentation changes needed
- USWDS - Dependencies: update gulp-mocha
   - uswds PR: https://github.com/uswds/uswds/pull/5680
   - Changelog: N/A - Dependency bump; no user-facing change
- USWDS - Dependencies: Remove experimental module work from core
   - uswds PR: https://github.com/uswds/uswds/pull/5673
   - Changelog: N/A - Dependency bump; no user-facing change
- USWDS - Add discussion template
    - uswds PR: https://github.com/uswds/uswds/pull/5785
    - Changelog: N/A - not a user facing change
- USWDS: POAM March '24
    - uswds PR: https://github.com/uswds/uswds/pull/5800
    - Changelog: N/A - Dependency bump; no user-facing change

### CODE REVIEW

File: pages/patterns/create-a-profile/pronouns.md [hunk 1]
```diff
@@ -112,7 +112,6 @@ Provide a text entry field that supports a rich array of special characters and
 {:.site-component-section-title}
 ## References
 - Ask users for gender or sex. (n.d.) Retrieved on July 19, 2022, from [https://design-system.service.gov.uk/patterns/gender-or-sex/](https://design-system.service.gov.uk/patterns/gender-or-sex/)
-- Asking about gender in online forms. (September 18, 2015) Retrieved on July 19, 2022, from [http://www.practicemakesprogress.org/blog/2015/9/18/asking-about-gender-on-online-forms](http://www.practicemakesprogress.org/blog/2015/9/18/asking-about-gender-on-online-forms)
```
Comment (amyleadem) [inline pages/patterns/create-a-profile/pronouns.md:115]: This is returning a 404 and causing an html proofer error. It appears as though the website is broken and I could not find a replacement link.

File: pages/documentation/showcase.md [hunk 2]
```diff
@@ -67,7 +67,6 @@ If your project is currently using USWDS and you would like to see it included i
 - [Federal Front Door](https://labs.usa.gov/)
 - [Federal Privacy Council](https://www.fpc.gov/)
 - [Federal Public Key Infrastructure Guides](https://fpki.idmanagement.gov/)
-- [Federalist](https://federalist.18f.gov/)
```
Comment (amyleadem) [inline pages/documentation/showcase.md:70]: > [!Note]
> We already have an entry for cloud.gov on this page, so I just removed this reference.

File: _next/02-introduction.md [hunk 3]
```diff
@@ -52,7 +52,7 @@ The current global pandemic has shown us that access to digital services is more
 
 To improve, we have to take a more holistic look at the fragmented ecosystem of the thousands of federal websites that currently exist, all of which offer different experiences for the public to navigate.
 
-We’re already making important strides forward. The growth of [GSA](https://gsa.gov) shared tools and services like the [U.S. Web Design System]({{ site.baseurl }}), [Federalist](https://federalist.18f.gov), and [login.gov](https://login.gov) have baked-in priorities like accessibility, continuity of experience, and designing for user needs.
+We’re already making important strides forward. The growth of [GSA](https://gsa.gov) shared tools and services like the [U.S. Web Design System]({{ site.baseurl }}), [cloud.gov Pages](https://cloud.gov/pages/), and [login.gov](https://login.gov) have baked-in priorities like accessibility, continuity of experience, and designing for user needs.
```
Comment (amyleadem) [inline _next/02-introduction.md:55]: > [!Note]
> Replaced the federalist links with links to cloud.gov Pages. We should confirm that this is the correct replacement and that we are referring to cloud.gov correctly.

File: README.md [hunk 4]
```diff
@@ -84,7 +84,7 @@ This documentation site showcases USWDS components by pulling in pre-built HTML
 
 ### Deployment and previews
 
-This site is deployed on [Federalist](https://federalist.18f.gov/), which automatically builds the public site whenever commits are pushed to `main`. Federalist also builds public previews for each branch pushed to GitHub.
+This site is deployed on [cloud.gov Pages](https://cloud.gov/pages/), which automatically builds the public site whenever commits are pushed to `main`. Federalist also builds public previews for each branch pushed to GitHub.
```
Comment (amyleadem) [inline README.md:87]: > [!Note]
> Replaced the federalist links with links to cloud.gov Pages. We should confirm that this is the correct replacement and that we are referring to cloud.gov correctly.

File: pages/patterns/create-a-profile/pronouns.md (hunk 1 above)
Comment (Miskeyceh94) [reply pages/patterns/create-a-profile/pronouns.md:115]: Biar aja saya miskin , soal nya saya gak curang. Emank kenapa kalo saya miskin , emnk kamu bisa memperkaya saya , harta mu cukup untuk beli beras , gak cukup untuk bayar negara haha

File: _data/changelogs/component-table.yml [hunk 5]
```diff
@@ -4,6 +4,30 @@ type: component
...
 
 items:
+  - date: NNNN-NN-NN
+    summary: Created the `.usa-table--sticky-header` variant to allow sticky positioning on table headers.
+    summaryAdditional: Also added the `$theme-table-sticky-top-offset` setting, which controls the top position of sticky headers.
+    affectsStyles: true
+    affectsSettings: true
+    githubPr: 5420
+    githubRepo: uswds
+    versionUswds: N.N.N
```
Comment (mahoneycm) [inline _data/changelogs/component-table.yml:14]: > [!IMPORTANT]
> Needs updated updated USWDS version number when we update dates

```suggestion
    versionUswds: 3.8.0
```

File: _data/changelogs/component-table.yml [hunk 6]
```diff
@@ -4,6 +4,30 @@ type: component
...
+  - date: NNNN-NN-NN
+    summary: Added the `$theme-table-background-color` setting.
+    summaryAdditional: Use this setting to declare the table background color at a theme level.
+    affectsStyles: true
+    affectsSettings: true
+    githubPr: 5420
+    githubRepo: uswds
+    versionUswds: N.N.N
```
Comment (mahoneycm) [inline _data/changelogs/component-table.yml:22]: > [!IMPORTANT]
> Needs updated updated USWDS version number when we update dates

```suggestion
    versionUswds: 3.8.0
```

**Comment by mejiaj** [comment]
## CircleCI 

Information regarding test errors.

### Pa11y

Desktop timeout crash, but mobile passed.

### `test_build` errors

Errors are related to a 404 to unpublished release `https://github.com/uswds/uswds/releases/tag/v3.8.0`.

**source:** [circleci report →](https://app.circleci.com/pipelines/github/uswds/uswds-site/6246/workflows/25b874d3-26c4-4e94-9dcb-2e91cf626845/jobs/25835/parallel-runs/0/steps/0-104)

**Comment by thisisdano** [comment]
All tests pass locally. Merging over pa11y crashes on CircleCI

**Pull request merged**